project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -592,12 +593,17 @@ function c4m_user_og_c4m_visitor_cta($group) {
     ),
   );
 
+  $login_url = t('Please <a href="@login-page">log in</a> first if you have a C4D account', array(
```

Comment:
We never add markup inside a `t()` function.

please use the `l()` function to create a link.